services/src/main/java/org/keycloak/credential/WebAuthnPasswordlessCredentialProvider.java

```diff
@@ -57,4 +63,27 @@ public CredentialTypeMetadata getCredentialTypeMetadata(CredentialTypeMetadataCo
     protected WebAuthnPolicy getWebAuthnPolicy() {
         return getKeycloakSession().getContext().getRealm().getWebAuthnPolicyPasswordless();
     }
+
+    @Override
+    public CredentialMetadata getCredentialMetadata(WebAuthnCredentialModel credentialModel, CredentialTypeMetadata credentialTypeMetadata) {
```

Comment:
@adhamahmad I think that this is better in the parent class `WebAuthnCredentialProvider`. The transport message is OK for both webauthn credentials, so moving to the parent class is enough.